Unaccountable-Magpie/WorkforceManagement
# Departments List View

Purpose:
Shows the list of employees when the department button is clicked in the nav bar

How it fits in context of the project:
Departments are to be listed and made clickable for the details view (feature pending)
Specific Feature affected:
Departments list
[https://github.com/Unaccountable-Magpie/WorkforceManagement/issues/2](url)

How to test (Be thorough!):

1.Checkout into my branch ag-departments-view

2.Open BangazonApi.sln

3.Click the Execute button

4. Click departments on the nav bar

5. The list of departments should be shown and each one should be a link to the future details view as well as a Create button that goes nowhere as of now
